### ISSUE TITLE
Random t/065-Term-CLI-Command-Help.t failures

### ISSUE THREAD

**Issue by eserte** [sbakker/perl-Term-CLI#6]
On some of my smokers (older ones: FreeBSD 9.2, CentOS 6, Debian wheezy+jessie) one test randomly fails:
```
#   Failed test 'pager exit status propagates to status'
#   at t/065-Term-CLI-Command-Help.t line 108.
#   (in Term_CLI_Command_Help_test->check_pager)
# Looks like you failed 1 test of 24.
t/065-Term-CLI-Command-Help.t ........... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/24 subtests 
```

**Comment by sbakker** [comment]
Argh. I really though I had fixed it this time. :-( It seems that running external commands in unit tests is fraught with pitfalls.

**Comment by sbakker** [comment]
Found it. Even though I set `$SIG{PIPE}` to `IGNORE`, if the external script exits before the program manages to write to it, the `SIGPIPE` is recorded in `$!` and `$?` is set to `-1` rather than the exit status of the external program. Fixed by changing the "fake pager" to first drain `STDIN`.

Fixed in v0.04008
